### DISCUSSION TITLE
How to call with Python requests

### DISCUSSION THREAD

**Discussion by hulleyrob** [FlareSolverr/FlareSolverr#888] [General]
I tried adding it as a proxy but It seems that's not how it works, does anyone have a working sample?

**Comment by RextConnor** [comment]
import requests

proxies = {
   'http': 'http://proxy.example.com:port',
   'https': 'http://proxy.example.com:port',
}

url = 'https://examples.com'

response = requests.post(url, proxies=proxies)

make sure you are using the most recent requests module.

**Comment by hulleyrob** [comment]
Hey thanks for that I did try to post a solution I had found since asking but it wouldn't let me.
